release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1559,6 +1559,36 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
+
+* Previously, routes tied to the *Developer* perspective were not rendered. This produced 404 error messages when you refresh the Developer perspective. The current release fixes this issue by loading all the inactive routes. Now, the routes switch to the correct perspective before loading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929769[BZ#1929769])
+
+* Previously, the colors for the Low and Medium severity issues of the Image Manifest Vulnerabilities (IMVs) did not match the color representation shown in the (link:https://quay.io/[Quay.io]) interface. As a result, when the user changed the severity order of vulnerabilities to High, the IMVs ordered the issues incorrectly. This created confusion when reviewing the IMVs. The current release fixes this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[BZ#1942716])
+
+* Previously, if it encountered a conflict while updating a resource quota, the API server sometimes failed to create a resource and returned a 409 status code. In that case, the client had to retry the API request. With this update, when the API server encounters this type of conflict, it retries creating the resource up to three times. This is often sufficient to create the resource. Otherwise, the API server displays an error in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1920699[BZ#1920699])
+
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run.
+This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[BZ#1970470])
+
+* Previously, when users inserted a formatted YAML snippet in the YAML editor, the new selection did not match the new content in the snippet: the indentation was removed, and some random letters remained behind the selection. The current release fixes this issue. Now, the cursor remains in the position where it started and adds the missing indentation for the cursor end position. After inserting the YAML snippet, the new selection matches the new content. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952545[BZ#1952545])
+
+* Previously, annotations were passed to the specification of knative services along with metadata. As a result, decorators were shown for associated revisions of knative services in the topology. The current release fixes this issue by passing annotations only to knative service metadata. Now, decorators are shown only for knative services in the topology and not for associated revisions.
```

Comment:
Knative in all instance - K capital
only to knative service metadata./only to the knative service metadata.

Reply:
Done